README.md

```diff
@@ -100,7 +100,22 @@ To uninstall the chart:
 
 #### Deployment Probes Paramaters
 
+##### Startup Probe
+StartupProbe indicates that the Pod has successfully initialized. If specified, no other probes are executed until this completes successfully.
+
+| Name                     | Description                                                                                  | Value           |
+| ------------------------ | -------------------------------------------------------------------------------------------- | --------------- |
+| deployment.startupProbe.enabled | Enabled startup probe                                                                  | true       |
```

Comment:
I think be default it should be false. In most of the applications, startupProbes wont be used as liveness probes. it good to have but should not be enforced by default. user will have to provide extra parameters for its configuration

Reply:
seems like its false in values.yaml, just in README its wrong

Reply:
thanks, just updated the README to reflect the default values. enable is false, successThreshold and timeoutSeconds is not set use k8s default.